policyengine_canada/parameters/gov/provinces/ns/benefits/child_benefit/base.yaml

```
@@ -0,0 +1,9 @@
+description: Nova Scotia child benefit base amount
```

Similarly adjust other parameter descriptions (include period as well)
```suggestion
description: Nova Scotia provides this base amount for its child benefit.
```